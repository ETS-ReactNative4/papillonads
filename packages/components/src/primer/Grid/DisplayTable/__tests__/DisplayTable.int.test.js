import { renderDefault } from './DisplayTable.int.render'

describe('<DisplayTable />', () => {
  describe('Render', () => {
    test('must match renderDefault()', () => {
      expect(global.renderToJSON(renderDefault())).toMatchSnapshot()
    })
  })
})
