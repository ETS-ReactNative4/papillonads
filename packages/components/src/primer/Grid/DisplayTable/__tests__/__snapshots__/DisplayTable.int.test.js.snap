// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayTable /> Render must match renderDefault() 1`] = `
Array [
  <div
    className="d-table col-12 bg-blue-light"
  >
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      isSelected
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      date-created
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      date-modified
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      kind
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      name
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      size
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Sat May 07 2016
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Fri Sep 05 2014
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Internet security certificate
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      DTmnIjnVyaK.cer
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      760 terabyte (TB)
    </div>
  </div>,
]
`;
